lib/crypto/c_src/aead.c

```diff
@@ -190,6 +134,7 @@ ERL_NIF_TERM aead_encrypt(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
 ERL_NIF_TERM aead_decrypt(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
 {/* (Type,Key,Iv,AAD,In,Tag) */
 #if defined(HAVE_AEAD)
+    struct cipher_type_t *cipherp = NULL;
```

Comment:
In general, I think variables that aren't used in the error path should be uninitialized when defined.  That way the compiler will warn if you accidentally use it before initializing it to a real value.